sktime/utils/_testing/hierarchical.py

```diff
@@ -171,9 +171,11 @@ def _bottom_hier_datagen(
                     ["l" + str(i - 1) + "_agg"]
                 )["l1_agg"].transform(
                     lambda x: "l"
-                    + str(i)
+                    + str(i)  # noqa B023
                     + "_node"
-                    + "{:02d}".format(_sample_node(node_lookup.index, i, rng))
+                    + "{:02d}".format(
+                        _sample_node(node_lookup.index, i, rng)  # noqa B023
+                    )
                 )
```

Comment:
Ignored as I believe this is a [false alarm](https://github.com/PyCQA/flake8-bugbear/issues/269)

Reply:
I think so